app/src/main/java/io/left/meshim/database/MeshIMDao.java

```diff
@@ -40,6 +40,9 @@
     @Query("SELECT * FROM Users WHERE MeshID = :meshId")
     public abstract User fetchUserByMeshId(MeshID meshId);
 
+    @Query("UPDATE  Messages SET isRead = :val WHERE MessageID=:messageID ")
```

Comment:
the only time I  am using it when I load  messages in the chat activity. so messages are already there with an id. 
what error cases were you thinking?

Reply:
`isRead` not `IsRead` - surprised it didn't yell at you that that column didn't exist.

Reply:
Which is why I'm curious, since you _do_ call it, why it hasn't broken.

Reply:
it think its because the actual variable in the Message is still 
@ColumnInfo(name = "IsRead")
    private boolean isRead;
so it just references the actual variable.

Reply:
just for consistency I will change it  IsRead

Reply:
Perfect. If it _works_ then it works I guess, but yeah my gut said change it now so it doesn't inexplicably bite us later.